nixos/tests/systemd-initrd-luks-unl0kr.nix

```diff
@@ -1,75 +1,109 @@
-import ./make-test-python.nix ({ lib, pkgs, ... }: let
-  passphrase = "secret";
-in {
-  name = "systemd-initrd-luks-unl0kr";
-  meta = {
-    maintainers = [];
-  };
+import ./make-test-python.nix (
+  { lib, pkgs, ... }:
+  let
+    passphrase = "secret";
 
-  enableOCR = true;
+    debugPackages = with pkgs; [
+      coreutils-prefixed
+      toybox
 
-  nodes.machine = { pkgs, ... }: {
-    virtualisation = {
-      emptyDiskImages = [ 512 512 ];
-      useBootLoader = true;
-      mountHostNixStore = true;
-      useEFIBoot = true;
-      qemu.options = [
-        "-vga virtio"
-      ];
+      micro
+      nano
+    ];
+  in
+  {
+    name = "systemd-initrd-luks-unl0kr"; # How to run an interactive test: `nix run .#nixosTests.systemd-initrd-luks-unl0kr.driverInteractive'
```

Comment:
Please don't include comments that are explained in the test framework's documentation.